## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 467553`
Commit: `42d3faee36f24511df84165f1386951b64adc176`

---
### `aarch64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>ty</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>ty</summary>
<pre>  crates/ty_python_semantic/resources/mdtest/generics/specialize_constrained.md:199 unmatched assertion: revealed: ty_extensions.Specialization[T@constrained_by_gradual = Base]
  crates/ty_python_semantic/resources/mdtest/generics/specialize_constrained.md:199 unexpected error: 17 [revealed-type] "Revealed type: `ty_extensions.Specialization[T@constrained_by_gradual = object]`"

To rerun this specific test, set the environment variable: MDTEST_TEST_FILTER='specialize_constrain… - Creating a specializ… - Constrained typevar (23c2daffe4f76bb3)'
MDTEST_TEST_FILTER='specialize_constrain… - Creating a specializ… - Constrained typevar (23c2daffe4f76bb3)' cargo test -p ty_python_semantic --test mdtest -- mdtest__generics_specialize_constrained

--------------------------------------------------


thread 'mdtest__generics_specialize_constrained' (7216301) panicked at crates/ty_test/src/lib.rs:152:5:
Some tests failed.
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    mdtest__generics_specialize_constrained

test result: FAILED(B. 290 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 4.04s

error: test failed, to rerun pass `-p ty_python_semantic --test mdtest`</pre>
</details>
</details>

--- a/pkgs/by-name/ty/ty/package.nix
+++ b/pkgs/by-name/ty/ty/package.nix
@@ -14,14 +14,14 @@
 
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "ty";
-  version = "0.0.1-alpha.29";
+  version = "0.0.1-alpha.31";
 
   src = fetchFromGitHub {
     owner = "astral-sh";
     repo = "ty";
     tag = finalAttrs.version;
     fetchSubmodules = true;
-    hash = "sha256-64/SE+B5fIaPS2qWHBNza8w14ID9xUVH8GhXSSB8Fbw=";
+    hash = "sha256-aGsDeDlmd7ro3kI5YnpuMCByBQaOMsBWIbfgtRfkS4k=";
   };
 
   # For Darwin platforms, remove the integration test for file notifications,
@@ -35,7 +35,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
 
   cargoBuildFlags = [ "--package=ty" ];
 
-  cargoHash = "sha256-ySR1pe9zwihjTzr3qhv71LuJbrhBWgZ+z+n9qEEern4=";
+  cargoHash = "sha256-qVcK/4VjOM6vIBh+BBG7iK+x9RmoSlg8EurUKz+5xqs=";
 
   nativeBuildInputs = [ installShellFiles ];
 

